### ISSUE TITLE
Remove `Text` links from tab index in the editor

### ISSUE THREAD

**Issue by Skaiir** [bpmn-io/form-js#439]
__Describe the Bug__

![image](https://user-images.githubusercontent.com/17801113/204576511-dd728f81-9286-461d-a0cf-1dc9c83ddacf.png)

Currently, it is possible to tab to a link rendered by the markdown processor of Text in the editor view. The processor only really exists to provide a preview of what we're working on, and I don't think it makes sense to keep these links tab-able, especially as every other field is disabled in this view. 

This seems like a pretty simple thing to tackle.

Also optionally, I think it would also be good to make those links not receive mouse events to follow that same philosophy. None of the other components are interactive, and if they are it's most likely a bug. 

@pinussilvestrus @christian-konrad @vsgoulart @RomanKostka 
Happy to hear some thoughts here.

__Steps to Reproduce__

1. Open the form editor
2. Add a text field
3. Configure a valid link on it
4. Tab around

__Expected Behavior__

The link should be ignored by the tabbing order

**Comment by vsgoulart** [comment]
@Skaiir Wouldn't this prevent a keyboard-only user to check if the link is correct?

**Comment by christian-konrad** [comment]
@vsgoulart no, in the preview it will work, this is just for the edit panel

**Comment by christian-konrad** [comment]
@Skaiir good catch! Let's discuss it in on the next iteration plannings

(I suggest we could do a "spring cleaning", too?)
